_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -193,6 +195,17 @@ The `aria-label` property is [prohibited][] for an element with a `generic` role
 <div aria-label="Bananas"></div>
 ```
 
+#### Failed Example 4
+
+The `aria-checked` state is [prohibited][] for an `input type="checkbox"` element to avoid providing conflicting states.
```

Comment:
aria-checked is not prohibited for checkbox. This is a different type of problem, that should probably go into a separate rule.

Reply:
Hey Wilco
That's true, the definition of prohibited states and properties does not apply in this case.

Initially I thought there was not enough to make it a separate rule. I thought mainly of `aria-checked` and `aria-disabled` but things such as `aria-level` will also come into play in here and that could be a direct failure of WCAG.

In this case shouldn't we make a change to the **language feature** point of Expectation 1 to not include conflicting states and properties and write a separate rule?